ledger/ledger-api-client/src/it/scala/com/digitalasset/ledger/client/CommandClientIT.scala

```diff
@@ -153,7 +153,7 @@ final class CommandClientIT
       expectedMessageSubString: String,
   ): Future[Assertion] =
     submitCommand(submitRequest).map { completion =>
-      completion.getStatus should have('code (expectedErrorCode.value))
+      completion.getStatus should have(Symbol("code")(expectedErrorCode.value))
```

Comment:
symbol literals have been deprecated.

Reply:
No good reason, I just went for the mechanical transformation (regex replace \o/) instead of thinking about it as I should have. fixed in a33ebf455c78bb350594433fc466cde27b8056fd